src/librustc_mir/dataflow/impls/borrows.rs

```diff
@@ -90,6 +97,19 @@ impl<'a, 'tcx> Borrows<'a, 'tcx> {
                     borrows.insert(idx);
                 }
             }
+
+            fn visit_statement(&mut self,
+                               block: mir::BasicBlock,
+                               statement: &mir::Statement<'tcx>,
+                               location: Location) {
+                if let mir::StatementKind::EndRegion(region_scope) = statement.kind {
+                    error!("statement {:?} {:?}", statement, statement.source_info.span);
```

Comment:
Is this supposed to be in here? Or was this instrumentation you put in but was supposed to be removed before merging?